### ISSUE TITLE
Bug in deleting lines from selection only

### ISSUE THREAD

**Issue by Eldaw** [rintoj/blank-line-organizer#3]
I've just tried out the latest release, v0.1.2, to see if deleting blank lines only in the selection works properly. But it seems there is a problem.

First, here is my configuration:
```
"blankLine.insertLineAfterBlock": false,
"blankLine.keepOneEmptyLine": false,
"blankLine.languageIds": [],
"blankLine.triggerOnSave": false,
```

I have typed a few lines of text and then selected just some of them, like so:

![01_before](https://user-images.githubusercontent.com/3789849/29454498-6094b574-8405-11e7-81d5-5adb8b06b256.png)

Next, I have issued the command 'Remove blank lines'. This is the unexpected result:

![02_after](https://user-images.githubusercontent.com/3789849/29454555-859824dc-8405-11e7-9416-f6f5dcf96f23.png)

**Comment by Eldaw** [comment]
Hi @rintoj 

Did you have a chance to look at my bug reports yet?

**Comment by Eldaw** [comment]
Hi @rintoj, do you plan to release a new version of blank-line-organizer?

**Comment by rintoj** [comment]
@Eldaw, I can't reproduce this yet. Can you share a video of this issue?

**Comment by rintoj** [comment]
Hope you are using version `0.1.2`

**Comment by Eldaw** [comment]
Hi @rintoj, yes I'm using 0.1.2 as I mentioned in the first post. Here is an animated GIF to demonstrate:
![removeblank](https://user-images.githubusercontent.com/3789849/46816107-2aa29a80-cd74-11e8-9cd9-95f16d8dff11.gif)

**Comment by rintoj** [comment]
@Eldaw, thanks for sharing the video. I didn't realize you were selecting the line before running the command. I can see the issue here as well. I will push a fix today.

**Comment by Eldaw** [comment]
Thanks @rintoj. Sorry, I thought it was clear in the first post (and the title of this bug report, and in the screenshots) where I said the problem affects selections. Looking forward to the new version!

**Comment by rintoj** [comment]
@Eldaw, Fixed and published. Check `0.1.3`

**Comment by Eldaw** [comment]
It was fixed in 0.1.3 but unfortunately is broken again in 0.1.4.
